core/execution/future/market.go

```diff
@@ -167,6 +167,9 @@ type Market struct {
 	internalCompositePriceCalculator *common.CompositePriceCalculator
 
 	amm *amm.Engine
+
+	fCap   *types.FutureCap
+	capMax *num.Uint
```

Comment:
why this one, not use directly fCap.MaxPrice ?

Reply:
`fCap.MaxPrice` is the value in market precision, `capMax` is in asset precision.

Reply:
indeed